query_optimizer/tests/execution_generator/Select.test

```diff
@@ -656,7 +656,7 @@ SELECT SUM(EXTRACT(SECOND FROM value))
 FROM dates;
 --
 +-------------------------------+
-|SUM(EXTRACT(SECOND FROM value))|
+|SUM(EXTRACT(second FROM value))|
```

Comment:
@zuyu looked at https://docs.oracle.com/cd/B28359_01/server.111/b28286/intro002.htm
does not seem to indicate any about uppercase specification. However the examples given indicate capitalization